student_explorer/cron.py

```diff
@@ -0,0 +1,191 @@
+# -*- coding: future_fstrings -*-
+
+from __future__ import print_function #python 3 support
+
+from django.db import connections as conns
+
+import logging, datetime, json
+
+from django.conf import settings
+
+from django_cron import CronJobBase, Schedule
+
+from umich_api.api_utils import ApiUtil
+
+from seumich.models import Mentor
+
+import dateutil.parser as parser
+
+from decouple import config
```

Comment:
For the change below
```suggestion
from decouple import config, Csv
```